org.eclim/plugin.properties

```diff
@@ -1,4 +1,7 @@
 pluginVersion=${eclim.version}
 nailgun.server.host=127.0.0.1
 nailgun.server.port=9091
+http.server.enabled=true
```

Comment:
Makes sense - this was the initial reason why I introduced this flag -> I just forgot to set it to false by default.
Now changed -> See changes in the next commit.